languages/PHP.md

```diff
@@ -14,3 +14,20 @@
 efficient and concise API. It covers a lot of web developers needs: an ORM (named Eloquent) and DB
 migrations,  Authentication and Authorizations, templating with Blade, routing, Browser testing
 (named Laravel Dusk), API only (Lumen), assets compilation, etc. And all very well documented.
+
+---
+
+[**Code Igniter**](https://codeigniter.com/) CodeIgniter is a powerful PHP framework with a very small footprint, built for developers who need a simple and elegant toolkit to create full-featured web applications.
```

Comment:
Please, link to the repository, not to the website.

Reply:
Oh, im sorry. I will change it later. Im still in my school